modules/friendlytag.js

```diff
@@ -570,6 +578,24 @@ Twinkle.tag.updateSortOrder = function(e) {
 	if (alreadyPresentTags) {
 		alreadyPresentTags.forEach(generateLinks);
 	}
+
+	var statusElement = document.getElementById('tw-tag-status');
+	$('[name=articleTags], [name=alreadyPresentArticleTags]').click(function() {
+		if (this.name === 'articleTags') {
+			if (this.checked) Twinkle.tag.status.numAdded++;
+			else Twinkle.tag.status.numAdded--;
+		} else if (this.name === 'alreadyPresentArticleTags') {
+			if (this.checked) Twinkle.tag.status.numRemoved--;
+			else Twinkle.tag.status.numRemoved++;
+		}
+
+		// update statusElement text
```

Comment:
I think this'd be better served above the `getElementById` and with a little expansion
```suggestion
		// tally tags added/removed, update statusElement text
```